ibis/backends/datafusion/__init__.py

```diff
@@ -372,7 +373,7 @@ def _register(
             self.con.deregister_table(table_name)
             self.con.register_record_batches(table_name, [[source]])
             return self.table(table_name)
-        elif isinstance(source, pa.dataset.Dataset):
+        elif isinstance(source, ds.Dataset):
```

Comment:
This seems to be unrelated to including examples in the tests suit, just making sure you are ok with that.

Reply:
These are definitely related. They aren't hit unless the examples are run.